### ISSUE TITLE
Github for Windows Crashing

### ISSUE THREAD

**Issue by vazquezj** [desktop/desktop#8313]
This issue is occuring on Windows 10.

There are not a lot of details. I can open the app on Windows 10. After it opens, everything works fine. The moment I click on a text bar within the app, it crashes.

Steps:
1. Open App
2. Click Text Bar
3. App Crashes

It happens every time I open the app and try to type anything. Please let me know if you need more information. Thank you!

**Comment by billygriffin** [comment]
Hi @vazquezj, thanks for the issue and sorry you're having problems. 

Would you mind letting us know what version of Desktop, and what your OS version is? 

Also, could you upload a log file from a day that it occurred? You should be able to access that via `Help` -> `Show logs in Explorer`

Thanks!

**Comment by vazquezj** [comment]
My OS:
Edition - Windows 10 Pro
Version - 1903
OS Build - 18362.356

[2019-09-23.desktop.production.log](https://github.com/desktop/desktop/files/3642948/2019-09-23.desktop.production.log)

**Comment by steveward** [comment]
@vazquezj the first thing I would recommend trying would be a fresh installation of GitHub Desktop. 

Here's how to do that:

1. Close any running GitHub Desktop processes
2. Open File Explorer
3. Go to `%LocalAppData%` and delete the `GitHubDesktop` directory
4. Go to `%AppData%` and delete the `GitHub Desktop` directory
5. Reinstall GitHub Desktop

Let us know how that goes.

**Comment by vazquezj** [comment]
I opened the app setup and it crashes upon clicking the text bar. The exact same issue.

**Comment by steveward** [comment]
@vazquezj thanks for trying that. Could you look in `%LOCALAPPDATA%\CrashDumps` to see if there are any crash dumps for GitHub Desktop?

**Comment by vazquezj** [comment]
Yes, there are crash dumps. The files are not compatible with Github, so here is a screensho
![Screenshot_44](https://user-images.githubusercontent.com/31492885/65563980-66992600-df19-11e9-9679-89f994ec8d9e.png)
t.

**Comment by steveward** [comment]
Thanks for sharing that @vazquezj. There are a number of possible causes for the `0xC0000006` error code -- I did a quick search and found various Microsoft support threads listing various causes like anti-virus software, your machine having a virus, missing DLLs, or corrupted registry entries. Are you having problems with any other applications on this machine?

One thing you can try would be to open an elevated command prompt and run the `sfc /scannow` command to check your system for any errors. Let me know if that turns up anything.

**Comment by tierninho** [comment]
@vazquezj  As we have not heard back from you on this issue, we are closing it for now. Should you have any more details or questions, please reach out and we will be happy to help. Thank you.
